pkg/vpcmodel/abstractVPC.go

```diff
@@ -246,6 +246,20 @@ type RulesInTable struct {
 	RulesOfType RulesType
 }
 
+// RuleOfFilter a single rule in filter given the layer (SGLayer/NACLLayer)
+type RuleOfFilter struct {
+	LayerName  string             `json:"layer"`
+	FilterName string             `json:"table"`
+	RuleIndx   int                `json:"rule_index"`
+	RuleDesc   string             `json:"rule_description"`
+	IPBlocks   []*ipblock.IPBlock `json:"ip_blocks,omitempty"`
```

Comment:
what is the `IPBlocks` field? how is it related to the rule? missing some description

Reply:
The IPBlocks are the cidrs/IPBlocks the Rule refers to (src/dst/local/... however it is called in each filter).
Documentation added